.hilbishrc.lua

```diff
@@ -8,9 +8,9 @@ function doPrompt(fail)
 	))
 end
 
-print(ansikit.format('Welcome {cyan}'.. os.getenv 'USER' ..
-'{reset} to {magenta}Hilbish{reset},\n' .. 
-'the nice lil shell for {blue}Lua{reset} fanatics!\n'))
+print(ansikit.format('Welcome to {magenta}Hilbish{reset} {cyan}'.. os.getenv 'USER' ..
+'{reset}.\n' ..
```

Comment:
dont linebreak here

Reply:
![2021-04-07_080257-330x220_scrot](https://user-images.githubusercontent.com/17111639/113870664-af20d900-9777-11eb-9694-0af06cbc52b9.png)
That's how it was before

Reply:
![](https://modeus.is-inside.me/pCyOUNgJ.png)
use up the line smfh